packages/protocol/contracts/L1/libs/LibTokenomics.sol

```diff
@@ -107,5 +107,15 @@ library LibTokenomics {
             / (state.proofTimeTarget * state.adjustmentQuotient);
 
         blockFee = uint64(result.min(type(uint64).max));
+
+        // Keep it within 0.1 and 10 TKO and not allow proofTimeIssued accumulated
+        // so that fee could recover quicker when waiting is applied from provers.
+        if (blockFee < 1e7) {
```

Comment:
Shall we also cap the reward?

Reply:
Will this prevent us from identifying issues?

Reply:
I don't think so, it will just help stabilize things when proofs aren't being submitted at the normal pace. So either too fast or not at all when there's a network issue (problem with the prover, etc...). If that happens without the cap it would just take a longer time for things to get back to normal again.

Reply:
> Will this prevent us from identifying issues?

Or wait do you mean a cap on the fee or a cap on the reward? If on the reward, yes I think that could hide some issues.